app.py

```diff
@@ -28,13 +28,68 @@ def send_mail(form_msg, form_name, form_email):
     try:
         response = sg.client.mail.send.post(request_body=mail.get())
         # print("sent")
-    except urllib.HTTPError:
+    except urllib.error.HTTPError:
         print("not sent")
+        slack_notifier(mode=1)
     except Exception:
         print("Some other exception occured. Not sent")
+        slack_notifier(mode=2)
+
+    from_email = Email(os.environ['KOSS_EMAIL'])
+    # print("email init")
+    to_email = Email(form_email)
+    # print("email init2")
+    subject = "Query Recieved"
+    # print("subject")
+    content = Content("text/plain", "Hi "+form_name+",\n\n"+"Your message has been recieved by us and we \
+                       will respond to it soon."+"\nThank-you for communicating with us, have a nice day."+
```

Comment:
Please change this to :-  `Thank you for communicating with us. Have a nice day!`
While fixing the pep8 errors.